app/app.js

```diff
@@ -2,12 +2,16 @@ const express = require("express");
 const app = express();
 const { getTopics } = require("./controllers/topics.controllers");
 const { getEndpoints } = require("./controllers/endpoints.controllers");
-const { getArticleById } = require("./controllers/articles.controllers");
+const {
+  getArticleById,
+  getArticles,
+} = require("./controllers/articles.controllers");
 
 app.get("/api", getEndpoints);
 
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles", getArticles);
 app.get("/api/articles/:article_id", getArticleById);
 
 app.all("/api/*", (req, res, next) => {
```

Comment:
I think you're overly limiting your 404 path not found handler by doing `/api/*` what if someone requests '/banana'